### DISCUSSION TITLE
Vorta to manage remote borg backup target

### DISCUSSION THREAD

**Discussion by gaia** [borgbase/vorta#1044] [Ideas]
I'm on windows and I am covered with backups. But all my Linux machines use Borg. Can one use Vorta to manage remote Borg installations/targets?

**Comment by m3nu** [comment]
That's not really a use case we cover, but there is an open issue to give a custom `borg` binary path. Putting `ssh host borg` there could allow you to kinda manage remote backups.

**Reply by gaia** [reply]
mind pinging the issue so i can follow it? thx!
